pkg/api/v1beta1/awspcaissuer_types.go

```diff
@@ -36,7 +35,28 @@ type AWSPCAIssuerSpec struct {
 	Region string `json:"region,omitempty"`
 	// Needs to be specified if you want to authorize with AWS using an access and secret key
 	// +optional
-	SecretRef v1.SecretReference `json:"secretRef,omitempty"`
+	SecretRef SecretReference `json:"secretRef,omitempty"`
+}
+
+type SecretReference struct {
+	// Name is unique within a namespace to reference a secret resource.
+	// +optional
+	Name string `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
+	// Namespace defines the space within which the secret name must be unique.
+	// +optional
+	Namespace string `json:"namespace,omitempty" protobuf:"bytes,2,opt,name=namespace"`
+	// Specifies the secret key where the AWS Access Key ID exists
+	// +optional
+	AccessKeyIDSelector SecretSelector `json:"accessKeyIDSelector,omitempty"`
+	// Specifies the secret key where the AWS Secret Access Key exists
+	// +optional
+	SecretAccessKeySelector SecretSelector `json:"secretAccessKeySelector,omitempty"`
```

Comment:
Have you thought about including the `AWS_SESSION_TOKEN` key as well? It's a slim chance, but I could imagine that the values inside a secret are generated through an `sts assume-role`, and therefore would require this additional key.